rector.php

```diff
@@ -19,5 +20,6 @@
         SetList::CODE_QUALITY,
         LevelSetList::UP_TO_PHP_81,
         PHPUnitLevelSetList::UP_TO_PHPUNIT_100,
+        SymfonyLevelSetList::UP_TO_SYMFONY_54,
```

Comment:
:thinking: :thought_balloon: 

is the point of this to just check the (CLI)  Commands?

Reply:
Yeah and `symfony/filesystem` in tests.